code/modules/surgery/organs/liver.dm

```diff
@@ -64,3 +64,18 @@
 	icon_state = "pliver"
 	desc = "A large crystal that is somehow capable of metabolizing chemicals, these are found in plasmamen."
 
+/obj/item/organ/liver/cybernetic
+	name = "cybernetic liver"
+	icon_state = "liver-c"
+	desc = "An electronic device designed to mimic the functions of a human liver. It has no benefits over an organic liver, but is easy to produce."
+	origin_tech = "biotech=4"

```

Comment:
why do they need a downside? They're just supposed to be used for implantation into people who have lost a liver